articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+   - **Subscription** - This is your Microsoft Azure subscription.
+
+   - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - Select the location of the Azure resource.
+
+     ![Create new local network gateway](images/azs-public-browser-create-local-network-gateway-blade.png)
+
+5. Click **Create**.
+
+## Create the VPN connection in Public Azure
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your Azure Stack VPN:
+
+1. Navigate to your virtual network gateway by clicking **All services**, then select **Virtual network gateways** under the networking section.
+
+    ![Navigate to the Virtual network gateway](images/azs-public-browser-services-virtual-network-gateway.png)
+
+2. Select your virtual network gateway from the list.
+
+3. In the *new* blade for your virtual network gateway, select **Connections** under the **Settings** section.
+
+    ![Connections for virtual network gateway](images/azs-public-browser-virtual-network-gateway-connections-button.png)
+
+4. Click **Add**.
+
+5. In the **Add connection** blade, enter the following information:
+
+   - **Name** - The name of the connection.
+
+   - **Connection type** - Select **Site-to-site (IPSec)**.
+
+   - **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   - **Local network gateway**- Select the local network gateway you created earlier.
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - When adding a connection for an existing peer, the resource group can't be changed.
+
+   - **Location** - Select the location of the Azure resource.
+
+     ![Add new connection](images/azs-public-browser-add-connection-blade.png)
+
+6. Click **OK**.
+
+## Update the local Gateway IPs in Azure Stack and Public Azure
+
+In order to establish the connection you will need to identify the public IPs that you created in Azure Stack and Public Azure, and then update both local gateways.
+
+### Update the local gateway in Public Azure
+
+1. In the Azure Stack portal, navigate to the *connections* blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
+
+    ![All services connections networking section](images/azs-browser-all-services-connections-networking.png)
+
+2. Select the VPN connection you created in the earlier stages.
+
+    ![VPN connections](images/azs-browser-vpn-connection-view.png)
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
```

Comment:
Make a note of the public IP. You can find this in the *Virtual network gateway* section.